vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailActivity.kt

```diff
@@ -110,11 +116,20 @@ class RoomDetailActivity : VectorBaseActivity(), ToolbarConfigurable {
     companion object {
 
         const val EXTRA_ROOM_DETAIL_ARGS = "EXTRA_ROOM_DETAIL_ARGS"
+        const val EXTRA_ROOM_ID = "EXTRA_ROOM_ID"
+        const val ACTION_ROOM_DETAILS_FROM_SHORTCUT = "ROOM_DETAILS_FROM_SHORTCUT"
 
         fun newIntent(context: Context, roomDetailArgs: RoomDetailArgs): Intent {
             return Intent(context, RoomDetailActivity::class.java).apply {
                 putExtra(EXTRA_ROOM_DETAIL_ARGS, roomDetailArgs)
             }
         }
+
+        fun shortcutIntent(context: Context, roomId: String): Intent {
```

Comment:
It's annoying that this is required, but shortcuts can't have intents with parcelables :(

Reply:
Ok